python/foglamp/common/plugin_discovery.py

```diff
@@ -57,6 +62,36 @@ def get_plugin_folders(cls, plugin_type):
         else:
             return directories
 
+    @classmethod
+    def get_c_plugin_folders(cls, plugin_type):
+        directories = []
+        if os.path.isdir(utils._FOGLAMP_ROOT + '/cmake_build'):
```

Comment:
I would not include the cmake_build directories, this is not a recognised location in any way, just somewhere where somebody might build something.